webapp/src/main/java/life/qbic/views/MainLayout.java

```diff
@@ -2,142 +2,69 @@
+import com.vaadin.flow.router.Route;
+
 
 /**
  * The main view is a top-level placeholder for other views.
  */
+@PageTitle("Data Manager ")
+@Route(value = "data")
 public class MainLayout extends AppLayout {
 
-    /**
-     * A simple navigation item component, based on ListItem element.
-     */
-    public static class MenuItemInfo extends ListItem {
-
-        private final Class<? extends Component> view;
-
-        public MenuItemInfo(String menuTitle, String iconClass, Class<? extends Component> view) {
-            this.view = view;
-            RouterLink link = new RouterLink();
-            // Use Lumo classnames for various styling
-            link.addClassNames("flex", "h-m", "items-center", "px-s", "relative", "text-secondary");
-            link.setRoute(view);
-
-            Span text = new Span(menuTitle);
-            // Use Lumo classnames for various styling
-            text.addClassNames("font-medium", "text-s", "whitespace-nowrap");
-
-            link.add(new LineAwesomeIcon(iconClass), text);
-            add(link);
-        }
-
-        public Class<?> getView() {
-            return view;
-        }
-
-        /**
-         * Simple wrapper to create icons using LineAwesome iconset. See
-         * https://icons8.com/line-awesome
-         */
-        @NpmPackage(value = "line-awesome", version = "1.3.0")
-        public static class LineAwesomeIcon extends Span {
-            public LineAwesomeIcon(String lineawesomeClassnames) {
-                // Use Lumo classnames for suitable font size and margin
-                addClassNames("me-s", "text-l");
-                if (!lineawesomeClassnames.isEmpty()) {
-                    addClassNames(lineawesomeClassnames);
-                }
-            }
-        }
-
-    }
-
-    private AuthenticatedUser authenticatedUser;
-    private AccessAnnotationChecker accessChecker;
-
-    public MainLayout(AuthenticatedUser authenticatedUser, AccessAnnotationChecker accessChecker) {
-        this.authenticatedUser = authenticatedUser;
-        this.accessChecker = accessChecker;
+    protected Button register;
+    protected Button login;
 
-        addToNavbar(createHeaderContent());
+    public MainLayout() {
+        createHeaderContent();
     }
 
     private Component createHeaderContent() {
-        Header header = new Header();
-        header.addClassNames("bg-base", "border-b", "border-contrast-10", "box-border", "flex", "flex-col", "w-full");
-
-        Div layout = new Div();
-        layout.addClassNames("flex", "h-xl", "items-center", "px-l");
-
-        H1 appName = new H1("Data Manager");
-        appName.addClassNames("my-0", "me-auto", "text-l");
-        layout.add(appName);
-
-        Optional<TestUser> maybeUser = authenticatedUser.get();
-        if (maybeUser.isPresent()) {
-            TestUser testUser = maybeUser.get();
+        HorizontalLayout header = createHeaderLayout();
+        HorizontalLayout buttons = createHeaderButtonLayout();
```

Comment:
Java 17 way for local variables:
```suggestion
        var header = createHeaderLayout();
        var buttons = createHeaderButtonLayout();
```